karma.conf.js

```diff
@@ -0,0 +1,4 @@
+const webpackConfig = require('./webpack.config.js')
+const karmaDefaults = require('systematic').karma_get_defaults(__dirname, webpackConfig)
+
+module.exports = (karma) => karma.set(karmaDefaults)
```

Comment:
not sure this is necessary. Is it ? Maybe with an update of systematic to pickup the webpack config correctly ?

Reply:
yes it is dot files are not loaded correctly without it